spec/models/journal_entry_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+RSpec.describe JournalEntry, type: :model do
+
+  let(:journal) do
+    FactoryBot.create(:journal)
+  end
+
+  let(:entry) do
+    FactoryBot.build(:journal_entry, title: nil)
+  end
+  it 'is not valid without a title' do
+    expect(entry.valid?).to be_falsey
```

Comment:
Let's introduce the [gem](https://github.com/thoughtbot/shoulda-matchers) 'shoulda-matchers' to make the test cases more concrete.

Let's introduces tests for the model associations and attributes in this PR once you are done learning about this gem.

Reply:
Can we please leave this out for now, so i can correct the existing problems?